pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -8449,6 +8449,62 @@
   video_library_page_not_hacked_video_title_5: "How Not To Get Hacked: Websites and Wifi"
   video_library_page_not_hacked_video_title_6: "How Not To Get Hacked: Messaging and Downloads"
   video_library_page_not_hacked_video_title_7: "How Not To Get Hacked: Install Updates"
+  
+  about_us_page_title: "About Us"
+  about_us_page_hero_description: "Code.org® is an education innovation nonprofit dedicated to the vision that every student in every school has the opportunity to learn computer science as part of their core K-12 education. We expand access to computer science in schools, with a focus on increasing participation by young women and students from other underrepresented groups.  The leading provider of K-12 computer science curriculum in the largest school districts in the United States, Code.org also organizes the annual [Hour of Code](%{hour_url}) campaign, which has engaged more than 15% of all students in the world. Code.org is supported by generous donors including Microsoft, Amazon, Google and [many others.](%{others_url})"
+  about_us_page_section_title_1: "Over 80 million students and 2 million teachers on Code.org"
+  about_us_page_statistic_figcaption: "Code.org participation from the 2021-2022 school year. [See disaggregated data.](%{data_url})"
+  about_us_page_statistic_description: "Code.org increases diversity in computer science by reaching students of all backgrounds where they are — at their skill-level, in their schools, and in ways that inspire them to keep learning. The vast majority of the students on Code.org are from student groups historically underrepresented in computer science. Increasing diversity in computer science is foundational to our work, and [we encourage you to read more about our efforts](%{diversity_url})."
+  about_us_page_section_title_2: "Code.org in the News"
+  about_us_page_section_news_description: "See all past [news and announcements](%{news_url})"
+  about_us_page_section_table: "Our goals and metrics"
+  about_us_page_section_table_header_1: "Code.org Goal"
+  about_us_page_section_table_header_2: "Accomplishment"
+  abous_page_section_table_cell_1_1: "Improve diversity in CS *(US students)*"
+  abous_page_section_table_cell_1_2: "45% of Code.org students are young women, 50% are students from marginalized racial and ethnic groups, and 46% of US students are in high needs schools. Read more about [our approach to diversity.](%{diversity_url})"
+  about_page_section_table_cell_2_1: "Inspire students"
+  about_page_section_table_cell_2_2: "Tens of millions have tried the [Hour of Code](%{hour_url}). (426,370,561 served. 49% female)"
+  about_page_section_table_cell_3_1: "Reach classrooms"
+  about_page_section_table_cell_3_2: "591,636 teachers have signed up to teach our intro courses on [Code Studio](%{code_studio_url}) and 19,177,297 students are enrolled."
```

Comment:
Was it intentional that we now hardcode these numbers whereas [before](https://github.com/code-dot-org/code-dot-org/pull/53905/files#diff-cf71ff6d0ef1f7e19a35ce8b8b95efca41ddb40e534322514c3e24cb806419afL82) this change, we generated them programmatically?  And we translate these hardcoded values, too?

Reply:
Also, these specific values seem to be the out of date "fallback" values from [here](https://github.com/code-dot-org/code-dot-org/blob/3ce0dc5be8d75191b66905b589eed070df723c8e/pegasus/helpers/properties.rb#L48-L49), which are from 2017.  

From some quick delving on production, there seemed to be an issue preventing the newer values from being retrieved, but the actual values are much higher now.

Also, these are total numbers of students and teachers, so the "intro courses" nomenclature might be out of date.